pyproject.toml

```diff
@@ -16,8 +16,8 @@ dependencies = [
     "gwcs >=0.19.0",
     "numpy >=1.24",
     "astropy >=5.3.0",
-    "rad >=0.22.0, <0.23.0",
-    # "rad @ git+https://github.com/spacetelescope/rad.git",
+    # "rad >=0.22.0, <0.23.0",
+    "rad @ git+https://github.com/braingram/rad.git@source_catalog",
```

Comment:
Temporarily pointing to the source branch for https://github.com/spacetelescope/rad/pull/513 to pull in the required rad changes.